resources/lang/deu.ini

```diff
@@ -72,16 +72,16 @@ flags.natural = "Natürlich"
 ; tools
 ; wand tool
 tool.wand = "Auswahlwerkzeug"
-tool.wand.lore.1 = "Klicken sie mit der linken Maustaste auf einen Block, um die 1. Position einer Auswahl festzulegen"
-tool.wand.lore.2 = "Klicken sie mit der rechten Maustaste auf einen Block, um die 2. Position einer Auswahl festzulegen"
+tool.wand.lore.1 = "Klicke mit der linken Maustaste auf einen Block, um die 1. Position einer Auswahl festzulegen"
+tool.wand.lore.2 = "Klicke mit der rechten Maustaste auf einen Block, um die 2. Position einer Auswahl festzulegen"
 tool.wand.lore.3 = "Mit //togglewand können sie die Funktionalität umschalten"
 tool.wand.disabled = "Das Auswahlwerkzeug ist deaktiviert. Verwenden sie //togglewand, um es wieder zu aktivieren"
 tool.wand.setenabled = "Das Auswahlwerkzeug ist jetzt {%0}!"
 ; debug tool
 tool.debug = "Debug-Werkzeug"
-tool.debug.lore.1 = "Klicken sie mit der linken Maustaste auf einen Block, um Informationen zu erhalten"
-tool.debug.lore.2 = "wie z.B. den Namen und die Meta-Daten eines Blocks"
-tool.debug.lore.3 = "Mit //toggledebug können sie die Funktionalität umschalten"
+tool.debug.lore.1 = "Klicke mit der linken Maustaste auf einen Block, um Informationen zu erhalten"
+tool.debug.lore.2 = "wie z.B. den Namen und die Schadenswerte eines Blocks"
```

Comment:
I am thinking about changing back this line - i am unsure if the german users know what Schadenswerte means, but most know what "Damage"/"Meta values" are. I will ask some german friends for feedback - 
@SalmonDE @robske110 Was denkt ihr?

Reply:
I’d also keep “Damage” and “Meta-Values”, since they can be seen as technical terms in the minecraft world. And especially since most online resources describing what they are/do are in english, this should make things easier even for german users.

Reply:
I'll change line 83 back to "Meta-Daten" and  commit it.